src/xdocs/config_annotation.xml

```diff
@@ -194,6 +202,28 @@ public String getNameIfPresent() { ... }
   &lt;property name=&quot;allowSamelineParameterizedAnnotation&quot; value=&quot;true&quot;/&gt;
 &lt;/module&gt;
         </source>
+        <p>
+          Example to allow only one and only parameterized annotation on the same line
+        </p>
+        <source>
+@SuppressWarnings("deprecation") DataLoader loader; // no violations
+@Mock DataLoader loader; // violation
+        </source>
+        <p>
+          Use the default configuration with tokens:
```

Comment:
Please make this explanation in simpler terms that users can relate to.
Example: To configure the check to only validate annotations on methods.

Reply:
Done